src/main/java/reposense/parser/CsvParser.java

```diff
@@ -154,11 +155,16 @@ protected boolean isElementOverridingStandaloneConfig(final CSVRecord record, in
      * Returns the contents of {@code record} as a raw string.
      */
     private String getRowContentAsRawString(final CSVRecord record) {
-        StringBuilder inputRowString = new StringBuilder();
-        for (int colNum = 0; colNum < record.size(); colNum++) {
-            inputRowString.append(get(record, colNum)).append(",");
+        String emptyLineMessage = "[EMPTY LINE]";
```

Comment:
sounds good to me, this can be a constant though

Reply:
Done. I didn't do this initially as I felt it may clutter the class level constants.